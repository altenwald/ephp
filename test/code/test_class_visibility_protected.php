<?php

class Hello {
    protected $hello = "world";
}

$h = new Hello();
print "hello " . $h->hello . "\n";

