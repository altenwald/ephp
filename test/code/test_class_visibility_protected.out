
Fatal error: Cannot access protected property Hello::$hello in {{CWD}}/test/code/test_class_visibility_protected.php on line 8
